MRPT_doc (#44)
pyscf/pyscf.github.io: source/user/mrpt.rst

.. _user_mrpt:

Multi-Reference Perturbation Theory (MRPT)
******************************************

*Modules*: :mod:`mrpt`

Introduction
============

> The second-order perturbative energy correction can be added on top of the multireference wave function.

Reviewer: ```suggestion
A second-order perturbative energy correction can be added on top of a multireference wave function.
```

The MRPT module in PySCF supports the second-order N-electron valence state perturbation theory (NEVPT2) :cite:`Angeli2001NEVPT`  
using the strongly contracted (SC) internal contraction scheme, :cite:`Angeli2001SCNEVPT,Angeli2002` which is an intruder-state-free MRPT.

> SC-NEVPT2 can be applied to the CASCI/CASSCF wave function optimized by FCI and DMRG solvers. :cite:`Guo2016`

Reviewer: ```suggestion
SC-NEVPT2 can be applied to CASCI/CASSCF wave functions produced by the FCI or DMRG solvers. :cite:`Guo2016`
```

> Root_ID needs to be specified for the state-specific NEVPT2 calculation as ``mrpt.NEVPT(mc,root=Root_ID)``.
> (by default Root_ID=0 for the lowest root)

Reviewer: ```suggestion
The number of the CI root needs to be specified for state-specific NEVPT2 calculations with ``mrpt.NEVPT(mc,root=Root_ID)``.
By default, the NEVPT2 calculation is performed for the lowest root, `Root_ID=0`.
```

> A simple example (see :source:`examples/mrpt/03-dmrg_nevpt2.py`) of running
> SC-NEVPT2 calculations with FCI and DMRG solvers is 

Reviewer: ```suggestion
A simple example of SC-NEVPT2 calculations with the FCI and DMRG solvers is given in :source:`examples/mrpt/03-dmrg_nevpt2.py`:: 
```
